model/status_board/v1/owner_type.model

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Definition of a Status Board owner.
+class Owner {
```

Comment:
Does this type have its own `id`? If it doesn't then it should probably be a `struct` instead of a `class`. The difference between struts and classes is that classes are expected to have their own identity, and therefore the metamodel tool will automatically add the `id` attribute.

Reply:
Yes, it has its own ID.